### DISCUSSION TITLE
Allow excluding folders from search results

### DISCUSSION THREAD

**Discussion by Kartopod** [godotengine/godot-proposals#11313] [Editor]
When opening any 'quick open' menus, it would be great if you had the ability to exclude folders to not show up in the results. 
For example: Excluding the addons folder from the 'Quick open scene' and 'Quick open script' windows.
![image](https://github.com/user-attachments/assets/af814d2a-567d-4252-94a3-874ee7c310af)
This would declutter the list and improve the UX.

Perhaps even add a toggle to show all files, in case the user wants to search through the excluded files too.

**Comment by Calinou** [comment]
https://github.com/godotengine/godot/pull/56772 already handles this in 4.4 by having a toggle that excludes the `addons/` folder from search results.

**Reply by Kartopod** [reply]
That's a great improvement, thank you
